src/components/sparks/Particle.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+
+export interface ParticleCSSProperties extends React.CSSProperties {
+  '--particle-dir'?: number;
+  '--particle-offset'?: number;
+  '--index'?: number;
```

Comment:
What do you think about prefixing it like other vars `--particle-index` ?

BTW I can't see the usage of this var,  let's remove it then.

Reply:
Yes it is leftover, thx for catching